### PR TITLE
Add possibility to ignore "no data recorded" screen for one hour

### PR DESCRIPTION
**Pull request by tsteur**
I think it would be useful to ignore the "No data recorded" message at least for an hour. 

When users install Piwik or want to try it, then they might not be able to track data immediately making it hard for them to get an idea what Piwik is or can do. By being able to ignore this message a user can "click around" and explore Piwik even when not having any data tracked.

It may also help in the future for cases like https://github.com/piwik/piwik/issues/11568 in case we decide to show the no data message even though log data purging is enabled. Then even though there is currently no data recorded, the user would be still able to access previously generated reports. 

Feel free to suggest different wording.

### CODE REVIEW

File: plugins/SitesManager/lang/en.json [hunk 1]
```diff
@@ -76,6 +76,7 @@
         "SiteWithoutDataSetupTracking": "Please set up the %1$stracking code%2$s below into your website or mobile app if you haven't done already.",
         "SiteWithoutDataMessageDisappears": "This message will disappear as soon as some data was tracked for this website.",
         "SiteWithoutDataSetupGoals": "In the meantime, maybe you would like to %1$sset up some goals%2$s or learn more about the %3$sfeatures%4$s of Piwik in one of our %5$sguides%6$s or %7$sFAQs%8$s.",
+        "SiteWithoutDataIgnoreMessage": "Ignore this message for the next hour",
```
Comment (sgiehl) [inline plugins/SitesManager/lang/en.json]: Instead of `Ignore` maybe `Hide` or `Don't show` sounds a bit less negative

**Review by sgiehl — COMMENTED** [review]
I'm wondering if we maybe should make that option site dependent. So if someone hides the page and switches to another site without visits he would see it again.
But maybe that's not needed and otherwise it looks good to merge.

**Comment by tsteur** [comment]
Updated wording as suggested. Will update screens later and then merge. Think for now it's ok to not have it per site but can always change it later

**Comment by mattab** [comment]
Nice & original idea :+1: 

If possible please add it "per site". Feel free to merge either way

**Pull request merged**